modules/rhel-removing-rhcos.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-removing-rhcos-{context}"]
+= Removing RHCOS compute machines from a cluster
+
+After you add the Red Hat Enterprise Linux (RHEL) compute machines to your
+cluster, you can remove the the {op-system-first} compute machines.
+
+.Prerequisites
+
+* You have added RHEL compute machines to your cluster.
+
+.Procedure
+
+. View the list of machines and record the node names of the {op-system} compute
+machines:
++
+----
+$ oc get machines --namespace openshift-machine-api
```

Comment:
this command only works for Installer Provisioned Infrastructure (IPI) clusters, not work for User Provisioned Infrastructure (UPI) clusters.

$ oc get nodes -o wide
works for both Installer Provisioned Infrastructure (IPI) clusters and User Provisioned Infrastructure (UPI) clusters